packages/block-editor/src/hooks/style.js

```diff
@@ -127,7 +129,7 @@ const skipSerializationPathsEdit = {
  */
 const skipSerializationPathsSave = {
 	...skipSerializationPathsEdit,
-	[ `${ SPACING_SUPPORT_KEY }` ]: [ 'spacing.blockGap' ],
+	[ `${ BACKGROUND_SUPPORT_KEY }` ]: [ BACKGROUND_SUPPORT_KEY ], // Skip serialization of background support in save mode.
 };
```

Comment:
I'll look into this tomorrow, but I just noticed that if I switch off background image support at the block level, and have a block that already contains the attributes for background image, then the background image winds up being serialized to markup:

<img width="1272" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/4222f2c7-dcbd-494d-b4fb-a9ff29f45939">

In the case of the background image support, we want it to always be stripped from the saved markup, irrespective of whether or not the block supports the value.

Reply:
This is mostly resolved now, but I'll add a follow-up question below.